### ISSUE TITLE
Unit 6 Grammar

### ISSUE THREAD

**Issue by NickyWinder** [LanceGardner/The-Wolf-Garden#2]
do this one first plz. note new comments re format (in db)/basing it on Anna's units 1/2, etc?

**Comment by NickyWinder** [comment]
Have added early version

**Comment by NickyWinder** [comment]
[TB Chapter Fifteen v 1.docx](https://github.com/LanceGardner/The-Wolf-Garden/files/1150239/TB.Chapter.Fifteen.v.1.docx)

**Comment by NickyWinder** [comment]
[UNIT-6 LG.docx](https://github.com/LanceGardner/The-Wolf-Garden/files/1150616/UNIT-6.LG.docx)
[Chapter Fifteen v 1.1 + BK.docx](https://github.com/LanceGardner/The-Wolf-Garden/files/1150617/Chapter.Fifteen.v.1.1.BK.docx)

[TB Chapter Fifteen v 1.docx](https://github.com/LanceGardner/The-Wolf-Garden/files/1150619/TB.Chapter.Fifteen.v.1.docx)

**Comment by NickyWinder** [comment]
Not sure if either/both chap old 15 files will help you but they might. 
I've done all scaffolding and some answers etc for you - as many as I could.
I amended formatting doc in Dropbox a bit, btw.
The email re contacting M links to this chapter but doesn't affect your parts of it. Once your bits are done I'll delete the highlighting and address the prob bit.
I may have tomorrow off so will upload Unit 5 too as I think I did a bit to it, but doubt you'll do both in a day.
likin' the new system!
